sm64.ld

```diff
@@ -159,6 +159,7 @@ SECTIONS
    {
       KEEP(BUILD_DIR/asm/entry.o(.text*));
       KEEP(BUILD_DIR/asm/n64_assert.o(.text*));
+      KEEP(BUILD_DIR/asm/vc_bin.o(.text*));
```

Comment:
This can be guarded by an `#ifdef EEP` so that it doesn't use space in the main segment when eeprom isn't being used. `CPPFLAGS` is already passed to the linker script preprocessing step in the makefile.

Reply:
Done